ui/pages/swaps/swaps.util.ts

```diff
@@ -210,14 +210,21 @@ export async function fetchTopAssets(chainId: any): Promise<object> {
       fetchOptions: { method: 'GET', headers: clientIdHeader },
       cacheOptions: { cacheRefreshTime: CACHE_REFRESH_FIVE_MINUTES },
     })) || [];
+  const topAssetsList = response.filter((asset: { address: string }) =>
+    validateData(TOP_ASSET_VALIDATORS, asset, topAssetsUrl),
+  );
+  return topAssetsList;
+}
+
+// TODO: Replace `any` with type
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function fetchTopAssets(chainId: any): Promise<any> {
```

Comment:
It's just a string, right?

Reply:
For this function, I think you could just do this to improve the typing (ie intellisense):
```
export async function fetchTopAssets(
  chainId: string,
): Promise<Record<string, { index: string }>> {
  const response = await fetchTopAssetsList(chainId);
  const topAssetsMap = response.reduce(
    (_topAssetsMap, asset: { address: string }, index: number) => {
      return { ..._topAssetsMap, [asset.address]: { index: String(index) } };
    },
    {} as Record<string, { index: string }>,
  );
  return topAssetsMap;
}

Reply:
Good call, updated this